tracer/src/Datadog.Trace/DataStreamsMonitoring/Aggregation/BacklogBucket.cs

```diff
@@ -0,0 +1,18 @@
+// <copyright file="BacklogBucket.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
```

Comment:
Please add `#nullable enable` to the top of all new files in Datadog.Trace 🙂  (You don't need to add it for test files as it just adds noise there)

```suggestion

#nullable enable

```